### PR TITLE
More documentaion

### PR DESCRIPTION
**Pull request by yaacov**
**Description**

Add more inline docstrings.

### CODE REVIEW

File: pkg/client/connection.go [hunk 1]
```diff
@@ -15,7 +15,9 @@ limitations under the License.
 */
 
 // Package client contains the types and functions used to communicate with other services using
-// queues and destinations.
+// queues and topics.
+//
+// In this library we use the name destinations to describe both queues and topics.
```
Comment (oourfali) [inline pkg/client/connection.go]: term

File: pkg/client/message.go [hunk 2]
```diff
@@ -13,10 +13,22 @@ limitations under the License.
 
 package client
 
-// Message is an implementation of Message interface
+// A Message represents a message sent or received by a connection.
```
Comment (oourfali) [inline pkg/client/message.go]: Message represents

Comment (yaacov) [reply pkg/client/message.go]: :+1:  done

File: pkg/client/connection.go (hunk 1 above)
Comment (yaacov) [reply pkg/client/connection.go]: :+1: done

**Pull request merged**